iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -137,6 +145,7 @@
 
 public class HiveIcebergMetaHook implements HiveMetaHook {
   private static final Logger LOG = LoggerFactory.getLogger(HiveIcebergMetaHook.class);
+  private static final ObjectMapper jsonObjectMapper = new ObjectMapper();
```

Comment:
static vars / constants should be in upper registry

Reply:
Fixed.